js/bybit.js

```diff
@@ -2852,7 +2852,7 @@ module.exports = class bybit extends Exchange {
             'side': this.capitalize (side),
             'order_type': this.capitalize (type), // limit
             'time_in_force': 'GoodTillCancel', // ImmediateOrCancel, FillOrKill, PostOnly
-            'qty': amount,
+            'qty': this.amountToPrecision (symbol, amount),
```

Comment:
Note that amountToPrecision returns a string, so we might need `parseInt` here.

Reply:
@kroitor yeah but it is working with a string too 
![image](https://user-images.githubusercontent.com/43336371/171365585-0c924968-3e9c-4981-b759-fdc39dd2b867.png)

Reply:
@kroitor We can buy 2.5 of this contract, so I don't think we can use parseInt here, maybe leave it as is or parseFloat

Reply:
@carlosmiei we usually do `parseFloat (amountToString (` in such cases, despite that it works with an undocumented type. On a sidenote, we can probably add a boolean flag whether toPrecision-methods should return a number or a string.

Reply:
but like BTCUSDT on usdt perpetual can have decimals like 0.001 BTC amount as qty. Then integer won't work right ?

Reply:
@flieks BTC/USDT is a linear market so will be parsed as a float, check my last edit